src/vsr/superblock_free_set_encoded.zig

```diff
@@ -159,61 +157,69 @@ pub fn FreeSetEncodedType(comptime Storage: type) type {
             }
             set.* = .{
                 .write_block = set.write_block,
-                .checksum = vsr.checksum(&.{}),
```

Comment:
This was only required for one of the minor fuzzers which faked empty free set. We no longer need that, so I changed to `0`, to serve as "unitialized" state and to avoid confusing empty and not-yet-open free set.

Reply:
Would `undefined` work, or is it expected to be `0`?

Reply:
Technically, it will work! But I am wondering if I should use it here... If I set `checksum` to undefined, then, by the same token, I need to `undefined` all other fields, like `size`, and `block_count`, and what not. The init just allocates memory for all the fields, and we only learn the proper values of the fields in `open` and `checkpoint`.

But setting most of the things to `undefined` makes me uneasy --- I _think_ the behavior we'll get there would be implementation defined. Eg, one version of Zig sets it to `0xaa`, but another one  decides on `0x92`. 

So, if there _is_ a bug, and we read "unitialized" value in prod on some codepath, I think I'd prefer it to be deterministically zero, and rely on our asserts to catch us out. 

But I don't really have consensus with myself here, I can go either way! I guess the main conflict for me is that:

* `undefined` signals the right intention, and I'd love to use it for code clarity reasons.
* the runtime behavior of undefined is ... not sufficiently defined for my tastes!